src/python/visclaw/animation_tools.py

```diff
@@ -199,11 +199,15 @@ def make_html(anim, file_name='anim.html', title=None, raw_html='', \
     """
 
 
-    #html_body = anim_to_html(anim, fps=fps, embed_frames=embed_frames, \
-                 #default_mode=default_mode)
+    try:
+        html_body = anim.to_jshtml(fps=fps, embed_frames=embed_frames, \
+                                   default_mode=default_mode)
+    except:
+        print('*** anim.to_jshtml() failed, not making animation')
+        print('*** you may need to update your version of matplotlib')
```

Comment:
I was going through some unrelated code and started converting these types of print statements to warnings from [`warnings`](https://docs.python.org/3.8/library/warnings.html).  This will accomplish basically a similar thing except it can be filtered.  This is nice when a console is not available or if someone might want to actually raise a proper exception and either handle the problem or stop execution.